iiif_prezi3/helpers/to_reference.py

```diff
@@ -0,0 +1,14 @@
+from ..loader import monkeypatch_schema
+from ..skeleton import Manifest, Reference
+
+
+class ToReference:
```

Comment:
Based on your code here, I have reimplemented this to apply to all classes that can be turned into a reference in (#81), so I think that this file can be removed.

Reply:
it lived a short life, but shall be remembered...

Reply:
It has been reborn, stronger and more powerful than ever!

(insert emperor palpatine gif)